packages/cli/lib/projects.js

```diff
@@ -618,6 +616,32 @@ const logFeedbackMessage = buildId => {
   }
 };
 
+const createProjectComponent = async (component, name) => {
+  const i18nKey = 'cli.commands.project.subcommands.add';
+  let componentName = name;
+
+  const directoryPath = path.resolve(getCwd());
+  const configInfo = await getProjectConfig(directoryPath);
+
+  if (!configInfo.projectDir && !configInfo.projectConfig) {
+    logger.error(i18n(`${i18nKey}.error.locationInProject`));
+    process.exit(EXIT_CODES.ERROR);
+  }
+
+  const src = `${configInfo.projectConfig.srcDir}/`;
+
+  const componentPath = path.resolve(
```

Comment:
I think this should be:
```js
const componentPath = path.join(
  configInfo.projectDir,
  configInfo.projectConfig.srcDir,
  component.insertPath,
  componentName
);
```
My reasoning is
`configInfo.projectDir` is the absolute path to the project directory
`configInfo.projectConfig.srcDir` is the path to the src directory, relative to the project root dir
`component.insertPath` is the  path where the component should be inserted, relative to the src dir
`componentName` is the name of the component


We can use `path.join` here instead of `path.resolve` because of the fact that `configInfo.projectDir` is already an absolute path ([more context](https://stackoverflow.com/questions/35048686/whats-the-difference-between-path-resolve-and-path-join))